docs/installing.rst

```diff
@@ -49,9 +35,10 @@ Local Build and Install
 
    $ git clone https://github.com/ThreatResponse/aws_ir.git
    $ cd aws_ir
-   $ python setup.py
-   $ pip install dist/aws_ir-*.tar.gz
-   $ margaritashotgun -h
+   $ virtualenv env -p python3
```

Comment:
Should we be consistent and align this virtualenv instruction with the above? python3 -m virtualenv env